### PR TITLE
RHDEVDOCS-4483 odo dev versus odo deploy

### PR DESCRIPTION
**Pull request by gabriel-rh**
Version(s):
4.8+

Issue:
https://issues.redhat.com/browse/RHDEVDOCS-4483

Link to docs preview:
https://51340--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/developer_cli_odo/understanding-odo.html#odo-dev-deploy_understanding-odo

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

File: modules/odo-dev-deploy.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/understanding-odo.adoc
+
+:_content-type: CONCEPT
+[id="odo-dev-deploy_{context}"]
+= Develop and deploy
+
+The two most important commands in `odo` are `odo dev` and `odo deploy`. Sometimes, it can be hard to know which command to use.
```
Comment (jc-berger) [inline modules/odo-dev-deploy.adoc:9]: Consider rephrasing it so it's more user focused. What do you think of this rephrase? 

`...Learn when to use which command.` 
or
`...Identify when to use the correct command.` 

Something like that. What do you think?

**Review by jc-berger — CHANGES_REQUESTED** [review]

File: modules/odo-dev-deploy.adoc (hunk 1 above)
Comment (gabriel-rh) [reply modules/odo-dev-deploy.adoc:9]: Agreed - used first option

File: modules/odo-dev-deploy.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+== odo dev
+
+Use the `odo dev` command in the initial development phase of your application. For example, use `odo dev` when you are working with a local development environment and you are:
+
+- Constantly making changes
```
[the remark this answers is not in the document]
Comment (gabriel-rh) [reply modules/odo-dev-deploy.adoc:15]: Agreed - added - Making changes regularly

**Review by jc-berger — APPROVED** [review]
/lgtm

**Review by valaparthvi — COMMENTED** [review]
LGTM.

**Pull request closed without merging**